install_config/install/rpm_vs_containerized.adoc

```diff
@@ -13,24 +13,56 @@ toc::[]
 
 == Overview
 
-This section explores some of the preparation required to install {product-title}
-as a set of services within containers. This applies to hosts using either Red
-Hat Enterprise Linux or Red Hat Atomic Host.
+An RPM installation installs all services through package management and
+configures services to run within the same user space, while a containerized
+installation installs services using container images and runs separate services
+in individual containers.
 
-- For the xref:../../install_config/install/quick_install.adoc#install-config-install-quick-install[quick installation
-method], you can choose between the RPM or containerized method on a per host
-basis during the interactive installation, or set the values manually in an
+The default method for installing {product-title} on
+ifdef::openshift-origin[]
+Fedora, CentOS, or RHEL
+endif::[]
+ifdef::openshift-enterprise[]
+Red Hat Enterprise Linux (RHEL)
+endif::[]
+uses RPMs.
+
+[IMPORTANT]
+====
+When targeting a Red Hat Atomic Host system, the containerized method is the
+only available option, and is automatically selected for you based on the
+detection of the *_/run/ostree-booted_* file.
+====
+
+The following table outlines the differences between the RPM and Containerized
+methods:
+
+[options="header"]
+|===
+|Type |RPM  |Containerized
+
+|Installation Method |Packages via `yum` |Container images via `docker`
+|Service Management |`systemd` |`docker` and `systemd` units
+|Operating System | Red Hat Enterprise Linux | Red Hat Enterprise Linux or Red Hat Atomic Host
+|===
+
+[[install-config-install-quick-advanced-install]]
+== Containerized Installs: The Quick and Advanced Methods
```

Comment:
Yeah I played around with this a lot. I feel like every iteration didn't explain it well enough.

I'll go with "Installing on Containerized Hosts"